templates/content-certificate.php

```diff
@@ -38,19 +37,32 @@
 	</div>
 </div>
 
-<div class="llms-print-certificate no-print" id="llms-print-certificate">
-	<button class="llms-button-secondary" onClick="window.print()" type="button">
-		<?php echo _e( 'Print', 'lifterlms' ); ?>
-		<i class="fa fa-print" aria-hidden="true"></i>
-	</button>
+<?php
+if ( $cert->can_user_manage() ) {
+	$is_sharing_enabled = $cert->is_sharing_enabled();
+	?>
+	<div class="llms-print-certificate no-print" id="llms-print-certificate">
+		<button class="llms-button-secondary" onClick="window.print()" type="button">
+			<?php echo _e( 'Print', 'lifterlms' ); ?>
+			<i class="fa fa-print" aria-hidden="true"></i>
+		</button>
 
-	<form action="" method="POST">
-		<button class="llms-button-secondary" type="submit" name="llms_generate_cert">
+		<form action="" method="POST">
+			<button class="llms-button-secondary" type="submit" name="llms_generate_cert">
 			<?php echo _e( 'Save', 'lifterlms' ); ?>
-			<i class="fa fa-cloud-download" aria-hidden="true"></i>
-		</button>
+				<i class="fa fa-cloud-download" aria-hidden="true"></i>
+			</button>
+			<?php if ( get_post_type( $cert->get( 'id' ) ) === $cert->get( 'db_post_type' ) ) : ?>
+				<button class="llms-button-secondary" type="submit" name="<?php echo $is_sharing_enabled ? 'llms_disable_cert_sharing' : 'llms_enable_cert_sharing'; ?>">
```

Comment:
```suggestion
				<button class="llms-button-secondary" type="submit" name="llms_enable_cert_sharing" value="<?php echo ! $is_sharing_allowed; ?>">
```
Since <button>s can have a value why don't we use one var only?

Reply:
@eri-trabiccolo Actually nothing, i liked the idea :) 

i will do it and test it